OvmfPkg/IgvmSecureBootDxe/IgvmSecureBootDxe.inf

```diff
@@ -0,0 +1,43 @@
+## @file
+#  Enroll secure boot variables passed in via IGVM
+#
+#  Copyright (c) 2025, Red Hat Inc.
+#
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005

```

Comment:
Please use the most recent version here, and avoid the hex representation, i.e., just put `1.27`